opencog/README.md

```diff
@@ -54,7 +54,7 @@ You can use docker-compose for configuring your workspace on linux and Mac syste
     * cat /etc/hosts   # take note of the ip address for relex, e.g.
       `172.17.0.69     relex 8e7dc3a09f12 opencog_relex_1`
     * rlwrap telnet localhost 17001
-    * (define relex-server-host "172.17.0.69")
+    * (relex-server-host! "172.17.0.69")
```

Comment:
that is not right or there is some crazy conflicting definition somewhere.  The script `nlp/scm/processing-utils.scm` uses  `relex-server-host` as if it were a string that specifies the host name.  By default it is 127.0.0.1  here: `nlp/scm/nlp.scm`   perhaps you meant to say `(set! relex-server-host "172.17.0.69")`  which would change the default definition?